.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,16 @@
+name: Publish python package
+on:
+  push:
+    tags:
+      - "*.*.*"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
It is recommended to use actions/checkout@v2

Reply:
In v2  on tag event `git branch` don't show another branches